### ISSUE TITLE
Rules don't work with v-bind : short-syntax

### ISSUE THREAD

**Issue by extrem7** [vuejs/eslint-plugin-vue#2206]
Just don't react to `:` instead of `v-bind`

[Link to latest vue-eslint playground](https://ota-meshi.github.io/eslint-plugin-vue-demo/#eJxNzEsOgCAMBNCrNN0b9gQ8CRsFjCQIRIobw93FX3TZ15nZUUdjkaMguyQ/kO1VABDGbbB1owtGKpyK9wqBfS/eNM9xpYcF+9XbmfXqEkG2VFITHUMmOGdAwl5fuBYeEezutDTWA9e5M+s=)

**Comment by FloEdelmann** [comment]
Is that a valid shorthand though? I don't see it used in the Vue docs, e.g. here: https://vuejs.org/api/built-in-directives.html#v-bind

**Comment by extrem7** [comment]
Sure, It compiles to the same code

![image](https://github.com/vuejs/eslint-plugin-vue/assets/24792876/8dda22e5-4f06-4fc8-9442-30fc8abbb788)

**Comment by FloEdelmann** [comment]
Okay, I think we could consider supporting `:` then. However, since it is not documented at all, I'm not sure about this. @ota-meshi What do you think?

**Comment by ota-meshi** [comment]
I don't think we should use that shorthand for now. So I don't think we need any support.

https://github.com/vuejs/docs/pull/2294
